server/controllers/ContentExportImport.js

```diff
@@ -23,9 +22,12 @@ module.exports = {
     getContentByType: async (ctx) => {
         const contentTypeUid = ctx.request.query.uid;
         var populateSchema = '*'
-        if (contentTypeUid === 'api::simulation.simulation') {
-            populateSchema = SIMULATION_POPULATE
-        }
+        const customSchemas = strapi.plugin(PLUGIN_ID).config('customSchemas');
+        Object.keys(customSchemas).forEach(key => {
+            if (contentTypeUid === key) {
+                populateSchema = customSchemas[key];
+            }
+        })    
```

Comment:
if we have a config of type: 
customSchemas : {
   uid: string,
   customSchema: string[]
   },
   
  then a .find sounds better no ?